QCEP/qcep-7.md

```diff
@@ -0,0 +1,35 @@
+---
+qcep: 7
+title: Automatic PoSW Effective Difficulty Adjustment
+author: @qizhou, @junjiah
+status: Draft
+created: 2021-01-29
+---
+
+## Simple Summary and Motivation
+
+To better secure QuarkChain mainnet and enable bridging assets from other chains, we propose a self-adjusting mechanism to increase the network's hash power by lowering the effective difficulty for root chain PoSW stakers.
+
+The targeted network difficulty will match Ethereum mainnet's as of the time of writing, which is around 12.52 P (source: [2miners](https://2miners.com/eth-network-difficulty)). To compare, QuarkChain mainnet's root chain difficulty is ~91 T, roughly 1/100 of the former.
```

Comment:
The current diff is
92,980,657,535,890
which means the hash power is 92,980,657,535,890 / 60s / 1e12 = 1.54 Th/s